src/App.css

```diff
@@ -0,0 +1,26 @@
+.App {
+  /* text-align: center; */
+  min-width: 100%;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+.App-header {
```

Comment:
Ditto these - we might wanna go with something slightly more specific than `App` - but these I'm less concerned about

Reply:
We can actually go ahead and remove this selector.